projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -404,7 +408,7 @@ export class IgxDateRangePickerComponent extends DisplayDensityBase
     private get required(): boolean {
         if (this._ngControl && this._ngControl.control && this._ngControl.control.validator) {
             const error = this._ngControl.control.validator({} as AbstractControl);
-            return (error && error.required) ? true : false;
+            return error && error.required;
```

Comment:
Technically, the getter is now `boolean | undefined` like this

Reply:
reverted